pyro/distributions/relaxed_straight_through.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import, division, print_function
+
+import torch
+
+from pyro.distributions.torch import RelaxedOneHotCategorical
+from pyro.distributions.util import copy_docs_from
+
+
+@copy_docs_from(RelaxedOneHotCategorical)
+class RelaxedOneHotCategoricalStraightThrough(RelaxedOneHotCategorical):
+    """
```

Comment:
hope these docs come out looking the way they ought?

Reply:
if only there existed something like `make docs` for you to see them...